app/actions/GroupActions.ts

```diff
@@ -69,10 +69,10 @@ export function fetchAll() {
   });
 }
 
-export function fetchAllWithType(type: GroupType) {
+export function fetchAllWithType(type: GroupType | GroupType[]) {
   return callAPI({
     types: Group.FETCH,
-    endpoint: `/groups/?type=${type}`,
+    endpoint: `/groups/?type=${Array.isArray(type) ? type.join() : type}`,
```

Comment:
```suggestion
    endpoint: `/groups/`,
    query: type,
```
Might I suggest this

This should be handled by a `createQueryString` in the `callAPI`

Reply:
➕